src/createServer.js

```diff
@@ -1,11 +1,23 @@
 'use strict';
 
-// const express = require('express');
+const express = require('express');
+const { router: users } = require('./routes/users');
+const { router: expenses } = require('./routes/expense');
+const { userService } = require('./services/userService');
+const { expenseService } = require('./services/expenseService');
 
 function createServer() {
-  // Use express to create a server
-  // Add a routes to the server
-  // Return the server (express app)
+  const app = express();
+
+  app.use(express.json());
+
+  app.use('/users', express.json(), users);
+  app.use('/expenses', express.json(), expenses);
```

Comment:
You already used this above :)
```suggestion
  app.use('/users', users);
  app.use('/expenses', expenses);
```